aasm.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'generator_spec'
   s.add_development_dependency 'appraisal'
   s.add_development_dependency "simplecov"
-  s.add_development_dependency "codecov", ">= 0.1.10"
+  s.add_development_dependency "codecov", ">= 0.1.17", '< 0.1.20'
```

Comment:
This is temporary code waiting for https://github.com/aasm/aasm/pull/690 to be merged.
I will remove it before merge.

Reply:
#690 is merged now.

Reply:
Removed